Change 'Two images' pattern to wide width

enriquesanchez opened the pull request:
Updates the 'Two images' pattern to be wide width.

Before:
<img width="644" alt="Screen Shot 2020-07-09 at 19 54 24" src="https://user-images.githubusercontent.com/3276087/87105755-71f7dc80-c221-11ea-94aa-662bbca4be7c.png">


After:
<img width="1136" alt="Screen Shot 2020-07-09 at 19 54 37" src="https://user-images.githubusercontent.com/3276087/87105761-758b6380-c221-11ea-8a0f-17a6efe6ddb5.png">

One issue after this update is that the images are not high-res enough for this size, especially the left one.  Considering that we are constrained by the images we can source for the Core patterns library in Core, it might be OK to go with this until we come up with a solution re: images.

enriquesanchez committed:
Change gallery to wide width

kjellr commented:
> One issue after this update is that the images are not high-res enough for this size, especially the left one. Considering that we are constrained by the images we can source for the Core patterns library in Core, it might be OK to go with this until we come up with a solution re: images.

Hmm — this doesn't seem ok to me. We shouldn't include something pixelly... If we don't have any larger versions to draw from, maybe we swap with a different image from the Core patterns library for now? (if there are other good options there?)

enriquesanchez commented:
>Hmm — this doesn't seem ok to me. We shouldn't include something pixelly

@kjellr Thanks, and I think you're right. 

The other images we already use in the block and patterns library are also not high-res enough for this big of a size. While less noticeable, the pixelation is still there:

<img width="1138" alt="Screen Shot 2020-07-10 at 13 42 28" src="https://user-images.githubusercontent.com/3276087/87188804-170fc500-c2b5-11ea-9a19-b53da6188e20.png">

I'd be more than glad to donate a couple of images I've taken, but I worry this will lead to a larger discussion about sourcing images:

<img width="1143" alt="Screen Shot 2020-07-10 at 13 54 44" src="https://user-images.githubusercontent.com/3276087/87188975-66ee8c00-c2b5-11ea-8b5f-a4aa68511016.png">

mapk commented:
Those photos of the mountains are amazing, Enrique! But I too fear this will lead into another long discussion about whether those are the _right_ images. @kjellr what do you think. If we keep the lower rez images, we should keep the block set for the default width. If we want to go wider, we'll need new images.

kjellr commented:
I think the wider images are much nicer, but we definitely can't have them be pixelly. I'm sure we have higher-resolution versions of those same images — can we not upload those?

enriquesanchez commented:
I've requested to update the images for higher-res versions here: https://core.trac.wordpress.org/ticket/50650

desrosj committed:
Update two-images.php

Change the image URLs to reflect the proposed new location from the Core Trac ticket.

desrosj commented:
Since images cannot be replaced on the CDN reliably (caching is too persistent), the plan proposed on the Core Trac ticket is to use the same image names and just introduce them into a new folder, `core/5.5`. I updated this PR accordingly.

enriquesanchez commented:
Thanks for the update @desrosj! 🙏

@kjell @mapk New images from the wp.org CDN are higher res and all should look much better now:

<img width="1139" alt="Screen Shot 2020-07-16 at 19 05 10" src="https://user-images.githubusercontent.com/3276087/87734144-69f6ea00-c797-11ea-8289-364238804940.png">

mapk reviewed (approved):
The images work great! No pixelation. Should we update the pattern to be full width? Right now it's at wide. I'm approving just in case you decide to keep it at just wide. :)
